source/administration/changelog.md

```diff
@@ -4,6 +4,151 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
+ - Fixed an issue on Microsoft Edge (non-Chromium) where logging out caused the user to get stuck at a loading screen.
+ - Fixed an issue where a selected item in the Direct Messages **More** menu didn’t scroll into view when using keyboard navigation.
+ - Fixed an issue where users received ghost notifications when the "First name trigger mention" setting was set but the "First Name" was not set.
+ - Fixed an issue where post text was partially hidden by the post hover menu.
+ - Fixed an issue where users were unable to type color hex value into custom theme color input box.
+ - Fixed an issue where the badge with a mention count on the team sidebar did not increment when user was added to a channel.
+ - Fixed an issue where Group Message results were prioritized over Direct Message results for Full Name in the user autocomplete.
+ - Fixed an issue where the New Message indicator was broken when a webhook owned by the user posted to a channel.
+ - Fixed an issue where the active search bar was not vertically aligned with left edge of the right-hand side in tablet view.
+ - Fixed an issue where there were two scrollbars showing in the channel switcher.
+ - Fixed an issue where the "Start trial" message was unreadable in the System Console on dark theme on first load.
+ - Fixed an issue on Firefox where pasting an image also added the file as text.
+ - Fixed an issue where Python syntax highlighting handled ``"""`` strangely.
+ - Fixed an issue where formatting around inline codes was missing.
+ - Fixed an issue where ``GetPluginStatus`` didn't work in a non-cluster environment.
+
+### config.json
+Multiple setting options were added to `config.json`. Below is a list of the additions and their default values on install. The settings can be modified in `config.json`, or the System Console when available.
+
+#### Changes to Team Edition and Enterprise Edition:
+ - Under ``LdapSettings`` in ``config.json``:
+     - Added ``PublicCertificateFile``, to be able to upload the public certificate to be used for encryption with SAML configuration.
+     - Added ``PrivateKeyFile`` to be able to upload the private key to be used for encryption with SAML configuration.
+ - Under ``ServiceSettings`` in ``config.json``:
+     - Added ``EnableAPIChannelDeletion``, to permanently delete channels for compliance reasons.
+     - Added ``EnableAPIUserDeletion``, to permanently delete users for compliance reasons.
+ - Under ``NotificationLogSettings`` and ``ExperimentalAuditSettings`` in ``config.json``:
+     - Added ``AdvancedLoggingConfig``, to enable configuration options for setting audit targets.
+ - Under ``AnnouncementSettings`` in ``config.json``:
+     - Added ``AdminNoticesEnabled``, ``UserNoticesEnabled``, ``NoticesURL``, ``NoticesFetchFrequency``, and ``NoticesSkipCache``, to enable in-product notices to make users and Admins aware of the newest product enhancements from within Mattermost.
```

Comment:
We do not need to add these to the changelog since they are only for internal testing

AnnouncementSettings.NoticesURL
AnnouncementSettings.NoticesFetchFrequency
AnnouncementSettings.NoticesSkipCache

Reply:
@amyblais ^